src/components/formio/radio.tsx

```diff
@@ -51,11 +54,36 @@ export const Radio: React.FC<RadioProps> = ({
   options,
   label,
   required = false,
+  isPreview = false,
   tooltip = '',
   description = '',
 }) => {
+  const {values, setValues} = useFormikContext<ExtendedComponentSchema>();
+
+  const clearRadioValues = () => {
+    const updatedValues = {...values};
+    updatedValues[name] = '';
+    setValues(updatedValues);
+  };
+
   return (
     <Component type="radio" field={name} label={label} tooltip={tooltip} required={required}>
+      {values[name] && !isPreview && (
+        <div>
+          <button
+            type="button"
+            className="btn btn-secondary formio-button-remove-row"
+            onClick={clearRadioValues}
+          >
+            <i className="fa fa-times-circle" aria-hidden="true" />
+            <FormattedMessage
+              description="Clear radio values button label"
+              defaultMessage="Clear radio button values"
```

Comment:
"Clear selection" is probably shorter.

I think it's also better to put this button under the radio element, so that it doesn't move the radios when one gets selected